branch-name-style/action.yml

```diff
@@ -5,6 +5,19 @@ description: >
   Checks if the name of the branch follows the branch naming convention of
   PyAnsys.
 
+  .. note::
+
+      Branch names must start with one of the following prefixes:
+
+      - feat
+      - junk
+      - maint
+      - doc or docs
+      - no-ci
+      - testing
```

Comment:
Can we support test?

I am used to it 😄 .

Reply:
Sure!